EventFilter/EcalRawToDigi/src/MatacqRawEvent.cc

```diff
@@ -80,16 +80,13 @@ const MatacqRawEvent::field32spec_t MatacqRawEvent::side32              = {13,0x
 
 void MatacqRawEvent::setRawData(const unsigned char* pData, size_t maxSize){
   error = 0;
-  unsigned char* begin = (unsigned char*) pData;
-  int16le_t* begin16 = (int16le_t*) pData;
+  (const int16le_t*) begin16 = (const int16le_t*) pData;
```

Comment:
Please delete the first two parentheses. They are confusing and serve no purpose. The line should look like this:
`const int16le_t  *begin16 = (const int16le_t *) pData;`